addons/point_of_sale/tests/test_frontend.py

```diff
@@ -1137,6 +1137,19 @@ def test_multi_product_options(self):
         self.main_pos_config.with_user(self.pos_user).open_ui()
         self.start_tour("/pos/ui?config_id=%d" % self.main_pos_config.id, 'MultiProductOptionsTour', login="pos_user")
 
+    def test_shipping_date_on_receipt_tour(self):
+        if not loaded_demo_data(self.env):
```

Comment:
I don't think you need this. You can use the product created from the test class

Reply:
This is not for the product it is for the customers.